src/acl.c

```diff
@@ -2366,7 +2393,34 @@ sds ACLLoadFromFile(const char *filename) {
         ACLFreeUser(new_default);
         raxInsert(Users,(unsigned char*)"default",7,DefaultUser,NULL);
         raxRemove(old_users,(unsigned char*)"default",7,NULL);
-        ACLFreeUsersSet(old_users);
```

Comment:
it may be a good idea to obliterate / change ACLFreeUserAndKillClients (and make sure to solve it's usages differently) so that we don't have a risk of iterating all clients per user.

Reply:
I assume you meant get rid of `ACLFreeUsersSet`? 'Cause `ACLFreeUserAndKillClients` is still pretty useful in certain circumstances.